language_detector/main.py

```diff
@@ -1,9 +1,16 @@
-"""This is the entry point of the program."""
-
+# -*- coding: utf-8 -*-
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    best_guess = ''
+    best_guess_count = 0
+    for current_language in LANGUAGES:
+        count = len([words for words in text.strip().split(' ') if words in LANGUAGES[current_language]])
```

Comment:
On line 8 and 9, you should be using `languages` instead of `LANGUAGES` so that if the user provides a second argument to `detect_language`, the code will use that instead. When the user doesn't provide a second argument to `detect_language`, `languages` will be set to be `LANGUAGES` as you can see by the code on line 5 `languages=LANGUAGES`.